aws-lc-rs/src/cipher.rs

```diff
@@ -163,7 +163,7 @@ pub enum OperatingMode {
 #[non_exhaustive]
 pub enum CipherContext {
     /// A 128-bit Initalization Vector.
-    Iv128(FixedLength<16>),
+    Iv128(FixedLength<AES_IV_LEN>),
```

Comment:
Maybe a constant called `128_BIT_IV_LEN`? May not be AES in the future which could be confusing.